Editor/Components/GuidedDataCollectorEditor.cs

```diff
@@ -94,6 +98,16 @@ public override void OnInspectorGUI()
 
         }
 
+        private void StepToTargetEnum(bool iterateToNext = true)
+        {
+            if (currentPhalangeIndex >= t.OrderOfCalibration.Count)
+            {
+                currentPhalangeIndex = 0;
+            }
+            t.TargetSegment = t.OrderOfCalibration[currentPhalangeIndex];
+            if(iterateToNext) currentPhalangeIndex++;
+        }
+
         private void StepEnum(int amt = 1)
```

Comment:
Is this being used anymore?

Reply:
yeah, in the buttons for go previous and go next. We could remove that functionality entirely, but it's still better than using the dropdown, since it causes the quest rendering to pause while the editor dropdown is open. We could also set up something nicer like fuzzy searching for a target phalange, not sure if that will suffer from the same issue or not

Reply:
In retrospect, it didn't make much sense to have previous and next when the order was defined. So the previous and next buttons now move back and forth in the defined order if provided, else uses the global list as before